src/Endpoints/Indexes.php

```diff
@@ -152,49 +157,26 @@ public function fetchInfo(): self
         return $this->fill($response);
     }
 
-    public function update($body): self
+    public function update($body): array
     {
-        $response = $this->http->put(self::PATH.'/'.$this->uid, $body);
-
-        return $this->fill($response);
+        return $this->http->put(self::PATH.'/'.$this->uid, $body);
     }
 
     public function delete(): array
     {
         return $this->http->delete(self::PATH.'/'.$this->uid) ?? [];
     }
 
-    // Updates
+    // Tasks
 
-    public function getUpdateStatus($updateId): array
+    public function getTask($uid): array
     {
-        return $this->http->get(self::PATH.'/'.$this->uid.'/updates/'.$updateId);
+        return $this->tasks->getIndexTask(self::PATH.'/'.$this->uid, $uid);
     }
 
-    public function getAllUpdateStatus(): array
+    public function getAllTasks(): array
```

Comment:
I think we decided to go for `getTasks`

Reply:
Yep but there is the function `getAllIndexes` and `deleteAllIndexes` so I change it to be consistent but there is also `stats` and `getSettings`.